### ISSUE TITLE
Chrome 80: badge icon not displaying privacy grade

### ISSUE THREAD

**Issue by t-bowersox** [duckduckgo/duckduckgo-privacy-extension#429]
## Description
Since the Chrome 80 update, the badge icon no longer displays the current site's privacy grade. Instead, it displays the DDG logo. If you click on the icon, however, you do still see the privacy grade and other expected info.

## Steps to Reproduce

1. Update Chrome to v80 or above (I was personally using 80.0.3987.87 when reporting this).
2. Visit any site.

**Expected behavior:**
The badge switches from the DDG logo to the privacy grade.

**Actual behavior:**
The DDG logo continues displaying.

**Versions**

- Extension: 2019.12.12
- Browser: Chrome
- OS: Windows and Mac

## Additional Information

Prior to the release of Chrome 80, the extension displayed the grade as expected using 2019.12.12.

In the extension debugging console, I did find several `InvalidStateError: The source image could not be decoded.` messages. Here's a screenshot after visiting npr.org, showing both the console and the unchanged badge icon:

<img width="874" alt="Screen Shot 2020-02-08 at 9 12 42 PM" src="https://user-images.githubusercontent.com/11789189/74094976-e0d8da00-4ab7-11ea-834e-0a3d97198e75.png">

**Comment by JetPlaneJJ** [comment]
When I tried to replicate this, the DDG logo appears for a second, then changes to the privacy logo. Sometimes the privacy logo shows up and stays correctly. 

![image](https://user-images.githubusercontent.com/31530784/74626134-9d0f5180-5103-11ea-8114-8f4b09a3f02e.png)

**Comment by JetPlaneJJ** [comment]
Also it seems like the icon resetting to ddg logo was intended when going to a new site unless the site is broken. Is this still a problem?

![image](https://user-images.githubusercontent.com/31530784/74628276-2033a600-510a-11ea-925e-a4e2323f8146.png)

**Comment by t-bowersox** [comment]
Hi @JetPlaneJJ -- I am still experiencing the issue on my end, across every site I visit (even duckduckgo.com). None of them are on the [broken site list](https://duckduckgo.com/contentblocking/trackers-whitelist-temporary.txt) referenced in the code, either.

The `InvalidStateError` seems to arise during the `loadImagePath()` function in `extensions::setIcon`. It does seem to be getting the corresponding image path for the site's grade, but the image itself can't be decoded for some reason.

I set a breakpoint on `extensions::setIcon` for line 33, which is where the error was logged to my console. There must be something wrong with the `imageData` object. Not sure if this is helpful, but here's what I'm seeing:

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/11789189/74670175-9ec62d00-5176-11ea-8f85-c37edf7693ec.png">

I did some research and it seems like this could be related to [createImageBitmap()](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/createImageBitmap) (line 18). Searching through Chromium's bugs, I did manage to find this: https://bugs.chromium.org/p/chromium/issues/detail?id=1049214. It seems like this is a Chrome bug, but it might not be fixed until v81/82? Not sure if anything can be done to the DDG extension to mitigate it in the meantime. 😕
